pace.coffee

```diff
@@ -202,7 +202,7 @@ class Bar
       @el = document.createElement 'div'
       @el.className = "pace pace-active"
 
-      document.body.className = document.body.className.replace /pace-done/g, ''
+      document.body.className = document.body.className.replace / pace-done/g, ''
```

Comment:
Looks like `/(^| )pace-done( |$)/g` that doesn't pass the `someclass pace-done someotherclass` case http://regexr.com/3dfos

`/(^pace-done ?)|( pace-done)/g` for comparison http://regexr.com/3dfop